Automattic/wp-calypso
# Media: Introduce "Uploaded to" and "Attached to" Section

#### Changes proposed in this Pull Request

* Includes details about where the file has been attached and to which post it was uploaded to

#### Testing instructions

You'll need to apply Automattic/jetpack#20408 for this to work!

Visit the Media Library and open a file, then verify that the "Uploaded to" is displayed neatly in the corner which accurately corresponds to the information stated in WP-Admin. You can test the "Attached to" section by including a file on multiple posts.

<img width="506" alt="Screenshot 2021-07-19 at 17 36 59" src="https://user-images.githubusercontent.com/43215253/126195519-e40214d3-fdd0-4535-8c0e-e81c2dacc430.png">

cc @BogdanUngureanu

Fixes #54647
